scipy/linalg/lapack.py

```diff
@@ -722,6 +765,30 @@
 }
 
 
+# Place guards against markdown problems with special characters
```

Comment:
Why markdown?

Reply:
Also, this might be worth taking up in numpy/f2py

Reply:
Ah, I meant numpydoc rst format. I'll change it.

Reply:
I am writing a simple PR for f2py already so we can get rid of this later.